modules/extensions/scripts/token_helper.template.sh

```diff
@@ -2,11 +2,14 @@
 # Copyright 2021 Oracle Corporation and/or affiliates.
 # Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl
 
-TOKEN_FILE=~/.kube/TOKEN
+CLUSTER=$5
+REGION=$7
+
+TOKEN_FILE=~/.kube/TOKEN-$CLUSTER
```

Comment:
I recommend using `$HOME` instead of `~` as the latter can break during shell expansion, most obviously under `zsh` when placed inside double quotes (which folks have a tendency to do.

Reply:
Done